src/react/components/toolbar/saveProject.tsx

```diff
@@ -1,7 +1,13 @@
 import { ToolbarItem } from "./toolbarItem";
+import { toast } from "react-toastify";
```

Comment:
what's the difference between toast and the messageBox class we have?

Reply:
A `toast` is a message that is display to the user typically in the corner of a page and goes away after a few seconds automatically.  An alert/message box is a block UI dialog that requires user interaction to dismiss.